medusa/purge_decommissioned.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import sys
+import traceback
+
+from medusa.monitoring import Monitoring
+from medusa.purge import purge_backups
+from medusa.storage import Storage
+
+
+def main(config):
+    monitoring = Monitoring(config=config.monitoring)
+
+    try:
+        logging.info('Starting decommissioned purge')
+        storage = Storage(config=config.storage)
+        # Get all nodes for cluster backups
+        all_backups = list(storage.list_cluster_backups())
+
+        # Get all nodes for latest cluster backup
+        latest_backup = [storage.latest_cluster_backup()]
+
+        # Get decommissioned nodes
+        decommissioned_nodes = get_decommissioned_nodes(all_backups, latest_backup)
+
+        for node in decommissioned_nodes:
+            logging.info('Decommissioned node backups to purge: {}'.format(node))
+            backups = storage.list_node_backups(fqdn=node)
+            (nb_objects_purged, total_purged_size, total_objects_within_grace) \
+                = purge_backups(storage, backups, config.storage.backup_grace_period_in_days, config.storage.fqdn)
```

Comment:
**Issue:** you're purging the local fqdn here, not the decommissioned node fqdn.